FormConfig/EmailConfig.php

```diff
@@ -22,7 +24,8 @@ public function __construct(RenderedSubmission $submission)
 
         $this->from = $message['from'];
         $this->subject = $message['subject'];
-        $this->body = preg_replace('/^&quot;|&quot;$/', '', $message['body']);
+        $this->body =(!empty($message['body'])) ? $this->sanitiseBody($message['body']) : '';
+        $this->attachments = (!empty($message['attachments'])) ? $this->sanitiseAttachments($message['attachments']) : [];
```

Comment:
attachments can be set to [] by default, only assign a value if $message['attachments'] is not empty?